near/contracts/wormhole/src/lib.rs

```diff
@@ -602,6 +602,7 @@ impl Wormhole {
         true
     }
 
+    #[private]
```

Comment:
If you make the migrate function private, can you still migrate?

Reply:
The migrate function is called automatically during a contract upgrade.  The Real question that keeps me up at night is... if it is not private, can people do any harm since it basically overwrites the contract state with itself..